tests/e2e/vsphere_volume_fstype.go

```diff
@@ -92,6 +92,14 @@ var _ = ginkgo.Describe("[csi-block-vanilla] Volume Filesystem Type Test", func(
 	ginkgo.It("[csi-block-vanilla-parallelized] CSI - verify invalid fstype", func() {
 		invokeTestForInvalidFstype(f, client, namespace, invalidFSType, storagePolicyName, profileID)
 	})
+
+	ginkgo.It("[csi-block-vanilla-parallelized] CSI - Windows - verify valid fstype", func() {
+		invokeTestForFstype(f, client, namespace, ntfsFSType, ntfsFSType, storagePolicyName, profileID)
+	})
+
+	ginkgo.It("[csi-block-vanilla-parallelized] CSI - Windows - verify invalid fstype", func() {
```

Comment:
csi-windows label should be there to run it only for windows. remove other label